docs/Installation.md

```diff
@@ -447,15 +447,31 @@ sed -i '' '/[[:<:]]engine[[:>:]]/ s/=.*/=MySQL/' /usr/local/etc/zonemaster/backe
 Install, configure and start database engine (and Perl bindings):
 
 ```sh
-pkg install mysql56-server p5-DBD-mysql
+pkg install mysql57-server p5-DBD-mysql
 sysrc mysql_enable="YES"
 service mysql-server start
 ```
 
-Initialize the database:
+MySQl requires that the password for root is reset before any operations can be run. The

```

Comment:
Perhaps the explanatory text could be made into a note to distinguish it more clearly from the instructions.